scripts/interactive_tests/interactive_test_util.js

```diff
@@ -140,11 +161,7 @@ exports.InteractiveTest = function(interactiveServerStartCmd, port) {
           }
         };
         return verifyAll(0);
-      }).fin(function() {
```

Comment:
`fin` is equivalent to `Promise.finally`, can we use that instead of `catch`ing?